Automattic/woocommerce-services
# Fix PHP tests

Fixes #2638.

## How to test

1. Run `composer run test` locally and note tests work.
2. See the checks in this PR's "PHP Tests" section pass.
